src/test/java/com/github/fppt/jedismock/comparisontests/scripting/EvalTest.java

```diff
@@ -294,4 +293,268 @@ public void evalUnpackTest(Jedis jedis) {
         assertThat(jedis.smembers("myset"))
                 .containsExactly("the", "quick", "brown", "fox");
     }
+
+    // cjson tests
+    @TestTemplate
```

Comment:
I think that this huge set of tests deserves its own class in the same package, say `CJsonTest`